### DISCUSSION TITLE
Apple Silicon Monterey Windows 10 21H1 BSOD

### DISCUSSION THREAD

**Discussion by stuba** [utmapp/UTM#3254] [UTM Help (macOS)]
Any ideas what might help I have tried multiple different combinations for system and cpu. In best case I manage to get BSOD with
stop code: SYSTEM THREAD EXPECTION NOT HANDELED
What failed: prl strg.sys

These images are exported from Parallels and I have tried to run same image on same UTM version on my intel based mac mini Monterey. It does boot and I do get desktop. So most likely image should be fine and problem is related to Apple M1 chip and UTM.

Any ideas how to debug? I did try with qemu 6.0.1 and 6.1.0, but no luck there. I'm not that familiar with some firmware limit exceeded errors.

<img width="410" alt="Screenshot 2021-11-04 at 10 48 56" src="https://user-images.githubusercontent.com/818066/140289813-ceb1557a-65f3-44f6-be41-0f7aa18354c2.png">

**Comment by conath** [comment]
If I understand correctly, you are trying to run a VM in UTM that was already installed using Parallels on another Mac? If so: Parallels and UTM have *very* different backend technology, so Windows can't handle this change and causes BSOD. I would recommend you re-install Windows in UTM. You can then add the existing hard disk image as another drive and access it in your new Windows VM.

Anyway, I will move this to the Discussion section since it's not a technical problem with UTM.

**Comment by stuba** [comment]
I do understand the differences, but like I said this image did boot well under Intel based mac os x UTM.

Kind of a problem is that this is domain joined container and it is not trivial to reinstall. Thanks for replying!
